Makefile

```diff
@@ -11,7 +11,7 @@ SOURCE_FILE_DIR =  $(ROOT_DIR)/code/$(PROJECT_NAME)
 AAR_FILE_DIR =  $(ROOT_DIR)/code/$(PROJECT_NAME)/build/outputs/aar
 
 checkstyle:
-		(./code/gradlew -info -p code/android-campaign-library checkstyle)
+		(./code/gradlew -info -p code/campaign checkstyle)
```

Comment:
This will help cover the changes by just updating the value for `EXTENSION-LIBRARY-FOLDER-NAME`
```
(./code/gradlew -p code/$(EXTENSION-LIBRARY-FOLDER-NAME) checkstyle)
```

Reply:
will fix with next pr 👍